cvat/apps/engine/serializers.py

```diff
@@ -1207,6 +1206,72 @@ class SubLabeledShapeSerializer(ShapeSerializer, AnnotationSerializer):
 class LabeledShapeSerializer(SubLabeledShapeSerializer):
     elements = SubLabeledShapeSerializer(many=True, required=False)
 
```

Comment:
It would be helpful to add a comment explaining why these custom serializers are used.